lisa/sut_orchestrator/azure/features.py

```diff
@@ -1409,24 +1410,62 @@ def create_setting(
         cls, *args: Any, **kwargs: Any
     ) -> Optional[schema.FeatureSettings]:
         raw_capabilities: Any = kwargs.get("raw_capabilities")
+        capabilities: List[SecurityProfileType] = [SecurityProfileType.Standard]
 
-        value = raw_capabilities.get("HyperVGenerations", None)
-        if value and "V2" in str(value):
-            return schema.FeatureSettings.create(cls.name())
+        gen_value = raw_capabilities.get("HyperVGenerations", None)
+        cvm_value = raw_capabilities.get("ConfidentialComputingType", None)
+        if gen_value and "V2" in str(gen_value):
+            capabilities.append(SecurityProfileType.Boot)
+        if cvm_value:
+            capabilities.append(SecurityProfileType.CVM)
 
-        return None
+        return SecurityProfileSettings(
+            security_profile=search_space.SetSpace(True, capabilities)
+        )
+
+    @classmethod
+    def on_before_deployment(cls, *args: Any, **kwargs: Any) -> None:
+        settings = cast(SecurityProfileSettings, kwargs.get("settings"))
+        if SecurityProfileType.Standard not in settings.security_profile:
```

Comment:
The setting should be only one value, not a list. It just compares if the security_profile is Standard or not.